components/teams-service/storage/postgres/migration/sql/10_upgrade_to_v2_migrations.up.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
```

Comment:
what's the difference between these two?

Reply:
ugh good catch. This one should have just been a rename of the other (added the `up`) but somehow both were preserved.